locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -4368,22 +4289,37 @@ msgstr "Bezirk und Ort"
 msgid "Edit district and location"
 msgstr "Bezirk und Verortung bearbeiten"
 
-#: meinberlin/apps/projects/forms.py:48 meinberlin/apps/projects/views.py:334
+#: meinberlin/apps/projects/forms.py:51 meinberlin/apps/projects/views.py:330
 msgid "Invite users via email"
 msgstr "Nutzer*innen via E-Mail einladen"
 
-#: meinberlin/apps/projects/forms.py:53 meinberlin/apps/projects/views.py:335
+#: meinberlin/apps/projects/forms.py:56 meinberlin/apps/projects/views.py:331
 msgid "Invite users via file upload"
 msgstr "Nutzer*innen aus Datei einladen"
 
-#: meinberlin/apps/projects/forms.py:54
+#: meinberlin/apps/projects/forms.py:57
 msgid "Upload a csv file containing email addresses."
 msgstr "Laden Sie eine CSV-Datei hoch, die E-Mail Adressen beinhaltet."
 
-#: meinberlin/apps/projects/forms.py:70
+#: meinberlin/apps/projects/forms.py:73
 msgid "Please enter email addresses or upload a file"
 msgstr "Bitte E-Mail Adressen eingeben oder Datei hochladen"
 
+#: meinberlin/apps/projects/forms.py:79
+msgid "Project topics"
+msgstr "Projektthemen"
+
+#: meinberlin/apps/projects/forms.py:80
+msgid "Add topics to your project."
+msgstr ""
+"Bitte ordnen Sie Ihrem Projekt 1-2 Themenfelder zu. Ihr Projekt kann in der "
+"Projektübersicht nach diesen Themen gefiltert werden."
+
+#: meinberlin/apps/projects/serializers.py:130
+#: meinberlin/apps/projects/serializers.py:315
+msgid "starts in the future"
```

Comment:
`Starts`?

Reply:
hmm, this is not new (it just moved), it was always lowercase, so I'd tend to leave it like it is